critiquebrainz/frontend/templates/macros.html

```diff
@@ -35,3 +35,11 @@
     <abbr title="{{ _('Submit and delete your votes.') }}">{{ _('Review votes') }}</abbr>
   {% endif %}
 {% endmacro %}
+
+{% macro entity_image_setter(id, is_event) %}
```

Comment:
So here I'd rather have `entity_type` argument and do comparison inside the macro, as needed.